modules/final_radiological_review/php/NDB_Menu_Filter_final_radiological_review.class.inc

```diff
@@ -31,53 +30,58 @@
  * @param unknown $val The raw value for the key
  *
  * @return string Human readable version of raw data
- *
  */
 function _makePretty($key, $val)
 {
-    if( $key=='Conflict'
-        || $key=="Conflict2"
-        || $key=="Conflict"
-    ) {
+    if ($key=='Conflict' || $key=="Conflict2" || $key=="Conflict") {
         // The table was created with a boolean, which mysql turns into a tinyint.
         // Print 'Yes' or 'No' instead of 1 or 0.
         if ($val == 1 || $val === '1' || $val === 'true') {
             return 'Yes';
-        } elseif ($val === "0" || $val === "false") { // PHP turns the tinyint back into a string ....
+            // PHP turns the tinyint back into a string ....
```

Comment:
This comment looks like it's moved to the wrong line. It looks like it's part of the elseif statement, not the end of the if statement.